web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -48,6 +48,12 @@ const WorkspaceMenu: React.FC<Props> = ({
       icon: <Icon icon="api" />,
       show: "both" as MenuShowType,
     },
+    {
+      label: t("My Integration"),
```

Comment:
I've changed this to plural in a couple places arlready in the PR. But...a user CAN install more than one integration at a time, right? If they can't then of course the singular form is better. Please update only if it makes sense!

Reply:
I was depending on the style file.
Changed to plural (makes more sense)